base_decorator.rb

```diff
@@ -0,0 +1,15 @@
+require './nameable'
+
+class Basedecorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super
```

Comment:
- [ ] To fix this issue in your project, please remove the `super` keyword from your initialize method. The initialize method calls the parent class's initialize method with the same arguments, but your parent class does not have an initialize method. This caused an error when I tested your project. Simply removing the `super` keyword will resolve this problem. :blush: 
    
  ![image](https://github.com/IsmaelMastronardi/school-library/assets/92303891/40118d40-a78d-4856-adcd-407bc93f5542)